src/scenes/game-scene.ts

```diff
@@ -2,47 +2,54 @@ import * as Phaser from 'phaser';
 
 import toggleDebug from '@/helpers/toggleDebug';
 import smoothMoveCameraTowards from '@/helpers/smoothMoveCameraTowards';
-import toggleMusic from '@/helpers/toggleMusic';
 import useLocalStorage from '@/helpers/useLocalStorage';
 import isDev from '@/helpers/isDev';
+import settingsMenu from '@/helpers/settingsMenu';
 
 import Parallax, { ParallaxNames } from '@/objects/Parallax';
 import Level, { LevelConfigType } from '@/objects/Level';
 
-import SpinText from '@/objects/SpinText';
 import Ben3 from '@/objects/entities/Ben3';
 import Bat from '@/objects/entities/Bat';
 import Tomato from '@/objects/entities/Tomato';
-import Ball from '@/objects/Ball';
 import Hedgehog from '@/objects/entities/Hedgehog';
 
+import Ball from '@/objects/Ball';
+import Skull from '@/objects/Skull';
+
 import Audio from '@/objects/Audio';
+import Text from '@/objects/Text';
 
 const parallaxName: ParallaxNames = 'supermountaindusk';
 
 const levelConfig: LevelConfigType = {
   tilesetPng: './level/tileset/demo-tileset.png',
-  tiledMapJson: './level/tiled-level/test-flat.json',
+  tiledMapJson: './level/tiled-level/test-bumpy.json',
+  // tiledMapJson: './level/tiled-level/test-flat.json',
+
+  // tilesetPng: './level/tileset/tileset1.png',
+  // tiledMapJson: './level/tiled-level/mapData1.json',
```

Comment:
Shall we remove these now? They were kinda placeholders to begin with really. the tileset was the first thing I found when doing the map stuff

Reply:
yeah can do... i mean its true we probs wont use them again... and if we do want to it in future - it can be got from git somehow

Reply:
removed